nni/contrib/compression/base/wrapper.py

```diff
@@ -36,7 +36,7 @@
 
 
 class ModuleWrapper(torch.nn.Module):
-    def __init__(self, module: torch.nn.Module, module_name: str, config: Dict[str, Dict[str, Any]] | None = None, \
+    def __init__(self, module: torch.nn.Module, module_name: str, config: Dict[str, List[Dict[str, Any]]] | None = None, \
```

Comment:
I think we don't need to modify the api here, just modify `ModuleWrapper.config` -> `ModuleWrapper.configs` to minimize the risk.